1-js/11-async/02-promise-basics/article.md

```diff
@@ -31,9 +31,9 @@ When the executor obtains the result, be it soon or late - doesn't matter, it sh
 - `resolve(value)` — if the job finished successfully, with result `value`.
 - `reject(error)` — if an error occurred, `error` is the error object.
 
-So to summarize: the executor runs automatically, it should do a job and then call either `resolve` or `reject`.
+So to summarize: the executor runs automatically. It performs a job and then calls either `resolve` or `reject`.
```

Comment:
Is the original variant wrong?

Reply:
Not very wrong. Probably wouldn't have triggered a PR from me (I hadn't read this article yet), but the changed version is a bit better. However, I would offer another alternative:

"...runs automatically and attempts to do a job. When it is finished with the attempt it calls `resolve` if it was successful or `reject` if there was an error."

If it's not too wordy.